src/v/kafka/server/handlers/configs/config_response_utils.h

```diff
@@ -0,0 +1,771 @@
+/*
```

Comment:
> This extracts the topic config specific code in describe_configs and
moves it into a separate file so that it can later be reused in the
CreateTopic handler.

it is disappointing that we are moving all this code into a header. do you think we could leave the code in the .cc file and then move whatever is going to use it over to the .cc file too? it's ok if it that's not the case!